rest_api/.jhipster/Subscription.json

```diff
@@ -27,10 +27,15 @@
       "fieldId": 1,
       "fieldName": "subscriptionId",
       "fieldType": "Long"
+    },
+    {
+      "fieldId": 2,
+      "fieldName": "topics",
```

Comment:
If we are storing a regex/wildcard as topic maybe we can name it as `topicFilterString`?